docs/src/docs/guielement/guielement-checks.adoc

```diff
@@ -18,15 +18,15 @@ and it's `width` and `height` are both greater than 0.
 element.isDisplayed();
 ----
 
-Checks if the element is displayed and if it's partially or `complete` visible
-in the scroll area of the Viewport.
+Checks if the element is displayed and if it's partially or `fully` visible
```

Comment:
I think you can remove the code markers around fully. In the version before it points to the parameter which had the same name.